lib/galaxy/datatypes/test/Si_lowercase.cell

```diff
@@ -0,0 +1,42 @@
+# CELL file written 11:14:42 (GMT-0.0) 17th February 2021 from run Si
+
+%block lattice_cart
+   ANG
+       5.47538171155659       0.530767321973160E-35  -0.131324260787022E-34
+      0.530767321973160E-35    5.47538171155659       0.313104997179767E-35
+     -0.131324260787022E-34   0.313104997179767E-35    5.47538171155659
+%endblock lattice_cart
+
+%block cell_constraints
+      1   1   1
+      0   0   0
+%endblock cell_constraints
+
+%block positions_frac
+   Si             0.000000000000000       0.000000000000000       0.000000000000000
+   Si             0.000000000000000       0.500000000000000       0.500000000000000
+   Si             0.250000000000000       0.250000000000000       0.250000000000000
+   Si             0.500000000000000       0.000000000000000       0.500000000000000
+   Si             0.250000000000000       0.750000000000000       0.750000000000000
+   Si             0.750000000000000       0.250000000000000       0.750000000000000
+   Si             0.750000000000000       0.750000000000000       0.250000000000000
+   Si             0.500000000000000       0.500000000000000       0.000000000000000
+%endblock positions_frac
+
+FIX_COM : true
+
+%block species_pot
+   Si      3|1.8|5|6|7|30:31:32
+%endblock species_pot
+
+SYMMETRY_TOL :     0.001000
+
+%block symmetry_ops
+# Symm. op. 1       E
+          1.000000000000000       0.000000000000000       0.000000000000000
+          0.000000000000000       1.000000000000000       0.000000000000000
+          0.000000000000000       0.000000000000000       1.000000000000000
+          0.000000000000000       0.000000000000000       0.000000000000000
+%endblock symmetry_ops
+
+kpoint_mp_grid :    2   2   2
```

Comment:
@elichad this is the problem, when you upload files Galaxy will by default add a newline at the end of text files. So when we compare the uploaded content with the test content the hash sum won't match. Add a regular newline here and the tests will pass.